python/core/qgsmargins.sip

```diff
@@ -138,90 +138,14 @@ class QgsMargins
 
 
 bool operator==( const QgsMargins &lhs, const QgsMargins &rhs );
-%Docstring
- Returns \c true if ``lhs`` and ``rhs`` are equal; otherwise returns \c false.
- :rtype: bool
-%End
```

Comment:
this would be easily checkable by the perl script as we know the class-depth we're in.

also, more generally, isn't it the operators that should not me documented in python?